app/models/frilans_finans_invoice.rb

```diff
@@ -58,6 +64,13 @@ def ff_approval_status_name(with_id: false)
     FFInvoiceStatuses.approval_status(ff_approval_status, with_id: with_id)
   end
 
+  def validate_job_frilans_finans_job
+    return if job&.frilans_finans_job?
```

Comment:
FrilansFinansInvoice#validate_job_frilans_finans_job performs a nil-check

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Nil-Check.md)